src/pybind/mgr/cephadm/registry.py

```diff
@@ -39,7 +39,10 @@ def get_tags(self, image: str) -> List[str]:
         headers = {'Accept': 'application/json'}
         url = f'https://{self.api_domain}/v2/{image}/tags/list'
         while True:
-            r = requests.get(url, headers=headers)
+            try:
+                r = requests.get(url, headers=headers)
+            except requests.exceptions.ConnectionError as e:
+                raise ValueError(f'{e}')
```

Comment:
```suggestion
                msg = f"Cannot get tags from url '{url}': {e}"
                raise ValueError(msg) from e
```
add a more contextual error?